DigitalLearningSolutions.Web/Startup.cs

```diff
@@ -262,6 +262,7 @@ private static void RegisterDataServices(IServiceCollection services)
             services.AddScoped<ICentreContractAdminUsageService, CentreContractAdminUsageService>();

```

Comment:
I see unused imports here too (unrelated to this ticket though)

Reply:
Might as well remove them too